cm.json/src/test/java/org/apache/felix/cm/json/io/impl/JsonSupportTest.java

```diff
@@ -190,8 +190,6 @@ public void testSeveralComments() throws IOException {
                 + "  \"a\" : 1,\n"
                 + "  \n"
                 + "  \n"
-                + "\n"
```

Comment:
This seems to be a change in behaviour as line numbers will change after comments are removed

Reply:
fixed to retain the linecount